### PR TITLE
Styling on CheckIn and a little bit of Map

### PR DESCRIPTION
**Pull request by joshhwuu**
Added styling according to the [figma](https://www.figma.com/design/LmsxQqR1p1iCC7pL3o1hJ2/UBCEA-Lounge-App?node-id=2-67&node-type=frame) for the overall main page, CheckIn component and a little bit of the Map component. Currently looks like:

image
image
image

### CODE REVIEW

File: client/src/app/check-in/add-user.tsx [hunk 1]
```diff
@@ -39,44 +39,57 @@ const AddUser = () => {
...
         />
-        <TextField
-          label="Notes"
-          name="notes"
-          value={addGamerData.notes}
-          onChange={handleInputChange}
+        <Button
+          onClick={handleSubmit}
+          label="Add Gamer"
+          className="rounded bg-[#3A6AAC] px-4 py-2 text-[#DEE7EC] hover:bg-blue-500"
```
Comment (Jaden51) [inline client/src/app/check-in/add-user.tsx:82]: Mentioned in Nic's PR but add a type `submit` to the button here. He added a prop to make this easy. This was mb for not implementing this originally.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (joshhwuu) [reply .vscode/settings.json:1]: vscode settings, should've been git ignored...

File: client/src/app/lounge/pc-station.tsx [hunk 3]
```diff
@@ -44,20 +44,41 @@ const PCStation: React.FC<PCStationProps> = ({ pc, isOccupied, onClick }) => {
   }, [pc.startedAt, pc.membershipTier]);
 
   const isTimeUp = timeRemaining === "Time Up";
+  const isDoubleHeight = pc.pcNumber === 9 || pc.pcNumber === 14;
+  const isCab = pc.pcNumber === 21;
+  const isCheckIn = pc.pcNumber === 20;
```
[the remark this answers is not in the document]
Comment (joshhwuu) [reply client/src/app/lounge/pc-station.tsx:49]: This part of the code is a little weird but I'm not sure the best way to approach it. The idea is that there are certain PCs that should be rendered differently, and I thought that would be best handled _within_ pc-stations. Maybe documentation could be better (ie explicitly state that pcs 14 and 9 must be rendered extra tall as per the design, 20 should be rendered as the check-in desk and 21 should be labelled the Cab). This specific logic I can't really see living anywhere else

File: client/src/app/lounge/map.tsx [hunk 4]
```diff
@@ -5,45 +5,65 @@ import { useState } from "react";
...
-  ) => {
-    setSelectedPC(pc);
-    setTimeRemaining(timeRemaining);
-    setIsOccupied(isOccupied);
-  };
+  // position for each i-th pc is [x, y]
+  const positions = [
+    [5, 5], // Position for PC 1
+    [5, 6], // Position for PC 2
+    [5, 7], // Position for PC 3
```
[the remark this answers is not in the document]
Comment (joshhwuu) [reply client/src/app/lounge/map.tsx:21]: I think the position array is a bit easier to move around, I'll move that to a different file and just load it here

**Review by Jaden51 — APPROVED** [review]
lgtm

**Pull request merged**